thinc_apple_ops/blas.pyx

```diff
@@ -3,18 +3,28 @@ cimport numpy as np
 import numpy
 
 
-cpdef np.ndarray gemm(float[:, ::1] A, float[:, ::1] B, bint trans1=False, bint trans2=False): 
+cpdef np.ndarray gemm(float[:, ::1] A, float[:, ::1] B, np.ndarray out=None,
+                      bint trans1=False, bint trans2=False):
```

Comment:
This is a case where it would have been nice to have keyword-only options for everything after `A` and `B` because it wouldn't change any calling code. But that wouldn't work with the `cpdef` definition, right.

Reply:
Yeah, sadly not 😢, would have been much nicer with kwargs-only.

Reply:
But I don't think there's any users calling this directly, right? For whom the switch in order could be confusing?

Reply:
I would assume not. On the other hand it's a publicly accessible function, so maybe we should move `out` to the end?

Reply:
Yea, that's what I'd been considering. I suppose you wanted to keep consistency with the other `gemm` definitions in Thinc but IIRC one does use kwd-only options, and for the other I suppose it would simply be good practice to call the functions with keyword names.

So maybe moveing it to the end would be the least bad option here and at least would have 0 possibility of breaking bwd-compat.

Reply:
Ok, fixed!